docs/install.md

```diff
@@ -0,0 +1,31 @@
+Installing the Pelican Platform
+===============================
+
+This document explains how a user can download and install the Pelican client. 
+
+Installation Steps:
+--------------------
+
+### Install the Pelican Platform Binary
+Navigate to the [Pelican release page](https://github.com/PelicanPlatform/pelican/releases/). Download the proper binary for the system you are running on and select which version you would like to install. 
+
+Our versions are built like so:
+For example: 7.1.2
+- 7 represents the latest major version release
+- 1 represents feature releases
+- 2 represents a bug fix/patch release
+
+### Extract the Binary
+Once the package has finished downloading, place it in your workspace and extract the binary
+
+### Test Functionality of the Pelican Platform
+Once extracted, make sure you are in the same directory as the **Pelican** executable. To test if everything works, we can do a simple **object copy** command:
```

Comment:
When pelican is installed with the rpm, deb or apk, it should automatically be added to the path, so we don't actually need to be in the same directory in those cases. When the binary is added to the path, we can just do `pelican <sub commands>` wherever we are, just like with other terminal commands.

Reply:
One possibility is "check to ensure that the pelican executable is on your path with `which pelican` "